# This file holds an array of type detected objects msg	
ObjectDetected[] objectArray